docs/migration-guides/sfa-ios-v8-to-v9.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: SFA iOS SDK - v8 to v9
+description: "SFA iOS SDK - v8 to v9 | Documentation - Web3Auth"
+sidebar_label: v8 to v9
+---
+
+This migration guide provides steps for upgrading from version v8 to v9 of the SFA iOS SDK. The
+guide outlines significant changes and enhancements.
+
+## Breaking Changes
+
```

Comment:
It should be mentioned that getTorusKey is now private, use connect instead. Then connected() can also be used to determine if connected.